README.md

```diff
@@ -70,14 +71,29 @@ Specify the upstream archive-data provider in your ``ape-config.yaml``:
 
 ```yaml
 ganache:
-  mainnet_fork:
-    upstream_provider: infura
+  fork:
+    ethereum:
+      mainnet:
+        upstream_provider: infura
 ```
 
-Otherwise, it defaults to the default mainnet provider plugin. You can also specify a ``block_number``.
+Otherwise, it defaults to the default mainnet provider plugin.
+You can also specify a ``block_number``.
 
 **NOTE**: Make sure you have the upstream provider plugin installed for ape.
 
 ```bash
 ape plugins add infura
 ```
+
+## Unlocking Accounts
+
+You can unlock / impersonate accounts at genesis time using Ganache.
+To do this, add the accounts to your config like this:
+
+```yaml
+ganache:
+  wallet:
+    unlocked_accounts:
```

Comment:
Would love to make this more clearly specify "Genesis" to describe the feature mord fully

Reply:
To me, it seems obviously genesis since it happens in the config file rather than mid-session.
But do you have a suggestion on naming? I can make it verbose like `unlocked_accounts_at_genesis` or something.

Reply:
How about `genesis_accounts`?

Reply:
one confusing thing about though is that it doesn't include the genesis accounts from the mnemonic.
it only includes the unlocked ones.

Reply:
`genesis_impersonated_accounts` so it is clear that it is not the normal genesis accounts, it is only the impersonated ones?

Reply:
`extra_genesis_accounts` ?

Reply:
Except I don't think they are really genesis accounts if you are using a forked network